components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-lifecycle/api-lifecycle.hbs

```diff
@@ -95,7 +95,26 @@
             </select>
         </div>
         <div class="col-md-3 col-sm-3">
-            <button id="update-tiers-button" type="button" class="btn btn-primary">Update</button>
+            <button id="update-tiers-button" type="button" class="btn btn-primary">{{i18n "Update"}}</button>
+        </div>
+    </div>
+    <div class="col-md-6 col-sm-6">
```

Comment:
I think you have to add relevant col-lg and col-xs classes as well. Otherwise there will be a responsive issue for there respective resolutions.